.github/workflows/changelog.yml

```diff
@@ -165,6 +165,8 @@ jobs:
             await createOrUpdateChangelogComment("", true);
       - name: Checkout changie config
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
An option here could be to selectively check out the change files/`.change` folder. I think the `filter` argument for the checkout action can be used for this.

That could help protect against this workflow being extended in future to run the code from the external PR (which would then be running with some write permissions).

Reply:
I will use `sparse-checkout` instead as I cannot work out what the pattern syntax is supposed to be for `filter` as there does not seem to be any docs or examples sadly.